x/stakeibc/keeper/icacallbacks_redemption_test.go

```diff
@@ -0,0 +1,212 @@
+package keeper_test
```

Comment:
Just noticed this [typo](https://github.com/Stride-Labs/stride/blob/main/x/stakeibc/keeper/icacallbacks_redemption.go#L47). Should be "RedemptionCallback". Wanna fix it in this PR?

Reply:
Where's the typo? Sorry, I think I'm missing it